apps/designer/app/designer/features/props-panel/use-props-logic.ts

```diff
@@ -60,19 +63,30 @@ const getRequiredProps = (
   const { component } = selectedInstanceData;
   const meta = getComponentMetaProps(component);
 
+  const props =
+    selectedInstanceData.props === undefined ||
+    selectedInstanceData.props.props.length === 0
+      ? []
+      : selectedInstanceData.props.props;
```

Comment:
if we just remove `|| selectedInstanceData.props.props.length === 0` will anything change?

Same below